src/App.js

```diff
@@ -0,0 +1,52 @@
+import ironhackLogo from "./images/ironhack-logo.png";
+import menu from "./images/menu.png";
+import icon1 from "./images/icon1.png";
```

Comment:
I would try to use more descriptive names so it's easier to maintain the application in the future :) 

```suggestion
import declarativeIcon from "./images/declarative.png";
```